src/meson.build

```diff
@@ -74,6 +74,7 @@ thorvg_lib = library(
 )
 
 thorvg_dep = declare_dependency(
+  compile_args: (lib_type == 'static') ? ['-DTVG_STATIC'] : [],
```

Comment:
Thanks, You can't just use `compiler_flags` here? Just wanna be sure.

Reply:
No, that includes all the build flags, like enabling AVX, etc., which the user doesn't need to copy.